bindings/python/iota_sdk/types/output.py

```diff
@@ -145,7 +256,7 @@ class OutputWithMetadata:
     """
 
     metadata: OutputMetadata
-    output: Output
+    output: AliasOutput | FoundryOutput | NftOutput | BasicOutput
```

Comment:
NIT: Maybe we could define alias type like 
`OutputType = Union[AliasOutput, FoundryOutput, NftOutput, BasicOutput]`, or smth like this, wdyt?

Reply:
`OutputType` would be conflicting with the enum class we have, do you have another name?

Reply:
`OutputVariant`? Idk, if it's too confusing, maybe we should leave it as is.

Reply:
`TreasuryOutput` is also not allowed here, so this would also not be a complete list of all possible outputs

Reply:
ok, So I think, we should leave it as is. More harm than good